.whatsnew.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "https://whatsnewapi.azurewebsites.net/schema",
+    "docSetProductName": ".NET",
+    "rootDirectory": "docs/",
+    "docLinkSettings": {
+        "linkFormat": "relative",
+        "relativeLinkPrefix": "/dotnet/"
+    },
+    "inclusionCriteria": {
+        "additionalMicrosoftOrgs": [
+            ".NET Platform"
+        ]
+    },
+    "areas": [
```

Comment:
Missing `azure` from the list?

**Note:** if these areas matches `dotnet-{area}` for ms.prod, I see that azure has the value of `azure-dotnet` not `dotnet-azure` like the others.

cc @BillWagner @scottaddie